net-p2p/qbittorrent/qbittorrent-9999.ebuild

```diff
@@ -4,20 +4,11 @@
 
 EAPI=6
 
-inherit flag-o-matic qmake-utils
+DESCRIPTION="BitTorrent client based on the Qt toolkit and libtorrent-rasterbar"
+HOMEPAGE="http://www.qbittorrent.org"
 
-DESCRIPTION="BitTorrent client in C++ and Qt"
-HOMEPAGE="http://www.qbittorrent.org/"
-
-if [[ ${PV} == *9999 ]]; then
-	inherit git-r3
-	EGIT_REPO_URI="https://github.com/${PN}/qBittorrent.git"
-else
-	MY_P=${P/_}
-	SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.xz"
-	KEYWORDS="~amd64 ~arm ~ppc64 ~x86"
-	S=${WORKDIR}/${MY_P}
-fi
+inherit git-r3
+EGIT_REPO_URI="https://github.com/${PN}/qBittorrent.git"
```

Comment:
I don't see all this advantage just because of the difference in the source. Unless someone actually lives with the git ebuild, and judging from the fact that qbittorrent ebuild hasn't seen any update from a long time (it's 3 versions old), I think the 9999 it's not used by anyone and could even been removed